src/auditor/dynamic_linkage.jl

```diff
@@ -1,4 +1,5 @@
 using ObjectFile.ELF
+using Patchelf_jll: patchelf
```

Comment:
I see BB2 is already using patchelf, which is great: https://github.com/JuliaPackaging/BinaryBuilder2.jl/blob/54202516e430264dc6060f763c55a7f0fbbd542a/BinaryBuilderAuditor.jl/src/Scanning.jl#L1.  But at a quick glance I don't see multi-threading enabled yet.